testsuite/features/step_definitions/command_steps.rb

```diff
@@ -71,6 +71,12 @@
   $server.run("curl --insecure --proxy '#{proxy}' --proxy-anyauth --location '#{url}' --output /dev/null")
 end
 
+Then(/^it should be possible to use the FTP server$/) do
+  # Use an URL in Provo after the release
```

Comment:
```suggestion
  # TODO: Use a URL in Provo after the release
```
It's easier to track our temporary "hacks" if we have them as a TODO :wink:

Reply:
right

Reply:
done